packages/app/features/home/screen.tsx

```diff
@@ -1,13 +1,50 @@
 import Navbar from 'app/ui/navbar'
 import BackgroundImage from 'app/ui/background-image'
 import Footer from 'app/ui/footer'
+import { Pressable, Text, View } from 'app/design-system'
 
 export function HomeScreen() {
   return (
-    <BackgroundImage>
+    <View tw="w-full h-full flex flex-col h-full">
+      <BackgroundImage />
       <Navbar />
-      {/* <MainWrapper /> */}
+      <View
+        tw="mx-auto text-center flex-1"
+        sx={{ width: [300, 600], mt: [24, 42] }}
+      >
+        <Text
+          tw="mt-4 text-center font-raleway"
+          sx={{
+            fontSize: [36, 64],
+            lineHeight: [40, 76],
+          }}
+        >
+          Music NFTs on Stellar
+        </Text>
+        <Text
+          tw="mx-auto text-center mb-9 max-w-[500px] font-raleway font-semibold tracking-0.5"
+          sx={{
+            fontSize: [16, 18],
+            lineHeight: 24,
+            marginTop: [38, 32],
+          }}
+        >
+          Upload, buy or sell music NFTs on the Stellar Network. Join a
+          community of beatmakers!
+        </Text>
+        <SignUpButton />
+      </View>
       <Footer />
-    </BackgroundImage>
+    </View>
+  )
+}
+
+function SignUpButton() {
+  return (
+    <Pressable tw="mx-auto bg-white border-2 border-black rounded-full py-3 px-8">
```

Comment:
I'd create a button component in the design system with the props like `title`, `variant`, `size` etc... I think there will be many similar buttons across the app.